packages/jellyfish-api-core/src/category/blockchain.ts

```diff
@@ -164,6 +164,37 @@ export class Blockchain {
     return await this.client.call('getrawmempool', [verbose], 'bignumber')
   }
 
+  /**
+   *
+   * Get all in-mempool ancestors for the given transaction as string[]
+   * @param {string} txId the transaction id in mempool
+   * @param {boolean} verbose false
+   * @returns {Promise<string[]>}
+   */
+  getMempoolAncestors (txId: string, verbose: false): Promise<string[]>
+
+  /**
+   *
+   * Get all in-mempool ancestors for the given transaction as MempoolTx object
+   * @param {string} txId the transaction id
+   * @param {boolean} verbose true
+   * @returns {Promise<MempoolTx>}
+   */
+  getMempoolAncestors (txId: string, verbose: true): Promise<MempoolTx>
+
+  /**
+   *
+   * Get all in-mempool ancestors for the given transaction as JSON object
+   * if verbose is true else as string[]
+   *
+   * @param {string} txId the transaction id
+   * @param {boolean} verbose true for json object, false for array of transaction ids
+   * @returns {Promise<string[] | MempoolTx>}
+   */
+  async getMempoolAncestors (txId: string, verbose: boolean): Promise<string[] | MempoolTx> {
```

Comment:
In your MD file, you wrote this. Here, you change the order of return type

`getMempoolAncestors (txId: string, verbose: boolean): Promise<MempoolTx | string[]>`

Reply:
Also, verbose is optional. 

`return await this.client.call('getmempoolancestors', [txId], 'bignumber')` works too

Reply:
As verbose is optional, this new finding affects how you should write your test and how you should design your function signature

Reply:
See https://github.com/DeFiCh/ain/blob/master/src/rpc/blockchain.cpp#L583
default = false